### PR TITLE
Bug 1126641 - Update Fira to 4.002

### PR DESCRIPTION
**Pull request by michaelwu**

### CODE REVIEW

File: FiraMono-4.002/LICENSE [hunk 1]
```diff
@@ -1,8 +1,12 @@
-Copyright (c) 2012-2014, The Mozilla Foundation and Telefonica S.A.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Sans.
 
-This Font Software is licensed under the SIL Open Font License, Version 1.1
-or any later version.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Mono.
```
Comment (jfkthame) [inline FiraMono-4.002/LICENSE]: It looks like the LICENSE file here has ended up with both Fira Sans and Fira Mono names in it.

File: FiraSans-4.002/LICENSE [hunk 2]
```diff
@@ -1,8 +1,12 @@
-Copyright (c) 2012-2014, The Mozilla Foundation and Telefonica S.A.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Sans.
 
-This Font Software is licensed under the SIL Open Font License, Version 1.1
-or any later version.
+Copyright (c) 2014, Mozilla Foundation https://mozilla.org/
+with Reserved Font Name Fira Mono.
```
Comment (jfkthame) [inline FiraSans-4.002/LICENSE]: Again, this names both Fira Sans and Fira Mono.

File: FiraMono-4.002/LICENSE (hunk 1 above)
Comment (n7s) [inline FiraMono-4.002/LICENSE]: The weight part of the font name doesn't need to be reserved.  (the FAQ has a little blurb on that: http://scripts.sil.org/OFL-FAQ_web#7fba289b)

Comment (n7s) [inline FiraMono-4.002/LICENSE]: The weight part of the font name doesn't need to be reserved  (the FAQ has a little blurb on that: http://scripts.sil.org/OFL-FAQ_web#7fba289b).

**Pull request merged**